code/modules/logging/log_entry.dm

```diff
@@ -0,0 +1,114 @@
+
+// Schema version must always be the very last element in the array.
+
+// Current Schema: 1.0.0
+// [timestamp, category, message, data, world_state, semver_store, id, schema_version]
+
+/// A datum which contains log information.
+/datum/log_entry
+	/// Next id to assign to a log entry.
+	var/static/next_id = 0
```

Comment:
So these IDs are unique within a round, but not across rounds?

And what happens if we overflow, is that possible?

Reply:
if we manage to get over 256k+ log entries I'll be shocked